rm_vk/rm_vk/Extension/UIImageView+Extention.swift

```diff
@@ -0,0 +1,22 @@
+// UIImageView+Extention.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Расширение для загрузки фото по ссылке
+extension UIImageView {
+    // MARK: - Public methods
+
+    func load(url: URL) {
+        DispatchQueue.global().async { [weak self] in
```

Comment:
```suggestion
        DispatchQueue.global().async { in
```
в DispatchQueue ненадо